src/server/worker/common/common.proto

```diff
@@ -16,7 +15,6 @@ message Input {
   string group_by = 6;
   bool lazy = 7;
   string branch = 8;
-  string git_url = 9 [(gogoproto.customname) = "GitURL"];
```

Comment:
Yeah, it has been unused since 2.0.  We deleted all things that read this field in 81144a5214e69492775d7fd52c7b48c359337130 Finish V2 / V1 merge (#5549).  (I did a git bisect with a regex.)  So consider this part of the pre-2.0 work that was un-done until now ;)